src/components/editable-input/editable-input.jsx

```diff
@@ -99,29 +100,40 @@ class InputEditable extends React.Component {
       </span>
     );
 
-    const renderButtons = this.state.editing ? (
+    const buttons = this.state.editing ? (
       <div className={classes.buttons}>
-        <Button variant="primary" modifier="action" onClick={this.onSubmit} disabled={this.props.hasError || this.state.saving}>
+        <Button
+          variant="primary"
+          modifier="action"
+          onClick={this.onSubmit}
+          disabled={this.props.disabled || this.props.hasError || this.state.saving}
+        >
           {this.state.saving ? <Spinner size={13} color="#FFF" /> : this.props.submitLabel}
         </Button>
-        <Button variant="secondary" modifier="action" onClick={this.onCancel} disabled={this.state.saving}>
+        <Button variant="secondary" modifier="action" onClick={this.onCancel} disabled={this.props.disabled || this.state.saving}>
```

Comment:
Nit pick: If you're going to line break the primary (which I love), why not line break this one as well? Far more readable imo.

Reply:
I agree completely, but I have not added the line breaks manually, I'm letting prettier handle that. It's a bit like the old problem we discussed previously, where prettier should be smarter about how it line breaks items such as this. I would much prefer if each value was on a separate line as well. I heard some talk about that prettier supposedly is getting smarter about this in newer versions, but since we're still stuck on the versions we are, we might have to live with this until we upgrade it.